includes/class-assets.php

```diff
@@ -77,6 +77,15 @@ public function register_common_scripts() {
         $this->register_select2();
 
 		wp_register_style( 'wpbdp-base-css', WPBDP_ASSETS_URL . 'css/wpbdp.min.css', array(), WPBDP_VERSION );
+
+		$custom_css = "
```

Comment:
Cool idea Paul! Is it possible to only include this with the plan price template where we use it?